content/04.results.md

```diff
@@ -253,7 +250,7 @@ A revision score ($y$-axis) close to 1 indicates that the LLM acting as a judge
 Each point represents the average score of paragraphs from a section in one of the four manuscripts: CCC, PhenoPLIER, BioChatter and Epistasis.
 ](images/llm_judge.svg "Automated assessments using LLM-as-a-Judge"){#fig:llm_judge width="90%"}
 
-The automatic assessment of paragraphs of different sections across four manuscripts is shown in Figure @fig:llm_judge.
-It can be seen that the two models used as judges, GPT-3.5 Turbo and GPT-4 Turbo, generally agreed and preferred the revised paragraphs over the original ones (revision score above zero) in most cases.
-The only section where the original paragraphs were clearly preferred was the Abstract of the PhenoPLIER and Epistasis manuscripts.
-GPT-3.5 Turbo preferred the original abstract of PhenoPLIER in most cases, and the model rationale (Supplementary File 5) is in line with our human assessment: the original abstract provides a more "detailed explanation" of the approaches and a "comprehensive overview of the research."
+The automatic assessment of paragraphs from different sections across four manuscripts is depicted in Figure @fig:llm_judge.
+It is observable that the two models used as judges, GPT-3.5 Turbo and GPT-4 Turbo, generally concurred and favored the revised paragraphs over the original ones (revision score above zero) in most instances.
```

Comment:
```suggestion
It can be seen that the two models used as judges, GPT-3.5 Turbo and GPT-4 Turbo, generally agreed and favored the revised paragraphs over the original ones (revision score above zero) in most cases.
```